lib/fastlane/plugin/cordova/actions/cordova_action.rb

```diff
@@ -28,17 +28,17 @@ def self.get_platform_args(params, args_map)
         args_map.each do |action_key, cli_param|
           param_value = params[action_key]
 
-          if action_key == 'build_flag' && param_value.kind_of?(Array)
+          if action_key.to_s == 'build_flag' && param_value.kind_of?(Array)
             unless param_value.empty?
               param_value.each do |flag|
-                platform_args << "--#{cli_param}=#{Shellwords.escape(flag)}"
+                platform_args << "--#{cli_param}=\"#{flag}\""
```

Comment:
doesn't like the Shellwords.escape?

Reply:
I will test without it and improve later. Because it escaped the = and thus didn't work

Reply:
it's now reapplied